fhir-persistence-schema/src/main/resources/add_any_resource.sql

```diff
@@ -44,6 +44,11 @@ BEGIN
 --  DECLARE CONTINUE HANDLER FOR NOT FOUND          SET v_not_found = 1;
   DECLARE CONTINUE HANDLER FOR c_duplicate        SET v_duplicate = 1;
 
+  -- Stop right here if we don't have a valid tenant
+  IF (fhir_admin.sv_tenant_id IS NULL) THEN
+  	SIGNAL SQLSTATE '99401' SET MESSAGE_TEXT = 'NOT AUTHORIZED: INVALID TENANT ID OR TENANT KEY';
+  END IF;
```

Comment:
The current code is designed to work without a sv_tenant_id.
Is this the behavior we want?

Reply:
set_tenant looks fine for me, I don't really know how to change set_tenant.sql only to achieve what we want...